src/date-picker/date-picker.jsx

```diff
@@ -43,6 +43,12 @@ const DatePicker = React.createClass({
     disableYearSelection: React.PropTypes.bool,
 
     /**
+     * Used to change the first day of week
+     * Defaults to 0 (Sunday)
```

Comment:
Can you remove this line? The documentation use `getDefaultProps` to display it automatically.